eclass/qt4-build-multilib.eclass

```diff
@@ -293,6 +293,8 @@ qt4-build-multilib_src_prepare() {
 qt4_multilib_src_configure() {
 	qt4_prepare_env
 
+	append-cxxflags -std=gnu++98
```

Comment:
please add a small comment, saying that Qt4 is not C++11/14 safe, and hence needs to be enforced for GCC 6 and above.